R/DataJoint.R

```diff
@@ -162,3 +162,25 @@ subset_and_add_grouping <- function(dat, groupings) {
     row.names(x) <- NULL
     x
 }
+
+
+#' Extract Observed Longitudinal Values
+#'
+#' Utility function to extract the observed longitudinal values from a [`DataJoint`] object
+#' @param object ([`DataJoint`])\cr The data used to fit a [`JointModel`]
```

Comment:
```suggestion
#' @param object ([`DataJoint`])\cr data used to fit a [`JointModel`].
```
(suggest to start argument descriptions lower case and end with full stop, at least for some parts of the package I used that convention)